src/main/java/spireMapOverhaul/zones/windy/patches/GoldRewardReductionPatch.java

```diff
@@ -0,0 +1,36 @@
+package spireMapOverhaul.zones.windy.patches;
+
+import basemod.abstracts.CustomSavable;
+import com.evacipated.cardcrawl.modthespire.lib.ByRef;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch2;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePrefixPatch;
+import com.megacrit.cardcrawl.rooms.AbstractRoom;
+
+public class GoldRewardReductionPatch {
+    public static int combatGoldReduction = 0;
+
+    //on the first call to addGoldToRewards, reduce gold reward param
+    @SpirePatch2(clz = AbstractRoom.class, method = "addGoldToRewards")
+    public static class GoldReductionPatch {
+        @SpirePrefixPatch
+        public static void patch(AbstractRoom __instance, @ByRef int[] gold) {
+            gold[0] = Math.max(0, gold[0]-combatGoldReduction);
```

Comment:
Hmm, since the gold reduction is based on the remaining health of the enemy which is in turn based on how much max health they have, this could result in pretty regular 0 gold rewards in act 3.

What do you think of either removing the reward in that case or capping it to at least 1 gold?

Reply:
I think removing the reward if it hits 0 gold seems like it'd be the most intuitive without needing extra explanation. A return statement if it hits 0 should work here.